db/c_test.c

```diff
@@ -126,6 +126,32 @@ static void CheckGetCF(
   Free(&val);
 }
 
+static void CheckPinGet(rocksdb_t* db, const rocksdb_readoptions_t* options,
+                        const char* key, const char* expected) {
+  char* err = NULL;
+  size_t val_len;
+  const char* val;
+  rocksdb_pinnableslice_t* p;
```

Comment:
I don't think this is a big problem. 

Btw, rocksdb_pinnableslice_t is just a typedef, the compiler may throw error:

```
error: variable has incomplete type 'rocksdb_pinnableslice_t' (aka 'struct rocksdb_pinnableslice_t')
  rocksdb_pinnableslice_t pin;
```